src/olympia/lib/settings_base.py

```diff
@@ -1332,7 +1337,12 @@ def read_only_mode(env):
 NODE_PACKAGE_JSON = path('package.json')
 NODE_PACKAGE_MANAGER_INSTALL_OPTIONS = ['--dry-run']
 
+# The manifest file is created in static-build but copied into the static root
+# so we should expect to find it at /<static_root/<static_build>/manifest.json
 STATIC_BUILD_PATH = path('static-build')
+# This value should be kept in sync with vite.config.js
+# where the manifest will be written to
+STATIC_BUILD_MANIFEST_PATH = path(STATIC_BUILD_PATH, 'manifest.json')
```

Comment:
A bit unfortunate that `vite` calls this `manifest.json`, same as webextensions manifest, but at least it doesn't conflict with django's `staticfiles.json` ...

Reply:
we can make the file whatever we want. I could make it "vite-manifest.json" or "banana.json" as long as the file names are aligned in both places it's up to us.

Reply:
https://github.com/mozilla/addons/issues/15326